src/missing_prices.py

```diff
@@ -57,7 +64,7 @@ def write_results(results: list[DuneTokenPriceRow], path: str, filename: str) ->
         os.makedirs(path)
     with open(os.path.join(path, filename), "w", encoding="utf-8") as file:
         for row in results:
-            file.write(str(row) + ",\n")
+            file.write(f"('{row[0]}', '{row[1]}', '{row[2]}', '{row[3]}', {row[4]}),\n")
```

Comment:
Really not a big fan of this. Let me know if you have any other ideas @bh2smith.

Reply:
Why did this need to change? Did you remove the __str__ implementation of the object? I am also not a fan.

Reply:
In solver rewards this is our recently introduced dune_repr

Reply:
This could be fixed by adding a custom `__repr__` for `Address` to `duneapi`

Reply:
Yea this address class isn't the best... I could push an update to that soon. Unless you wanna do it.

Reply:
If you're eager to merge this, just throw a todo in there